v3/src/components/graph/components/graph-attribute-label.tsx

```diff
@@ -65,8 +65,10 @@ export const GraphAttributeLabel =
       if (graphModel.pointsFusedIntoBars && place === secondaryPlace) {
         return t("DG.CountAxisView.countLabel")
       }
-      return attrIDs.map(anID => dataset?.attrFromID(anID)?.name)
-        .filter(aName => aName !== '').join(', ')
+      return attrIDs.map(anID => dataset?.attrFromID(anID))
+                    .filter(attr => attr?.name !== '')
+                    .map(attr => `${attr?.name} ${attr?.units ? `(${attr?.units})` : ""}`.trim())
```

Comment:
In the case of multiple attributes without units, I think this will result in extraneous spaces. I think it should be:
```typescript
                    .map(attr => `${attr?.name}${attr?.units ? ` (${attr?.units})` : ""}`.trim())
```